packages/manager/apps/pci-kubernetes/package.json

```
@@ -23,7 +23,7 @@
     "@ovh-ux/manager-config": "^8.0.2",
     "@ovh-ux/manager-core-api": "^0.10.1",
     "@ovh-ux/manager-core-utils": "^0.3.0",
-    "@ovh-ux/manager-pci-common": "^0.15.2",
+    "@ovh-ux/manager-pci-common": "^0.16.12",
```

Ooops, this won't work 😅

```suggestion
    "@ovh-ux/manager-pci-common": "^0.16.2",
```